temporal/api/cloud/sink/v1/message.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+package temporal.api.cloud.sink.v1;
+
+option go_package = "go.temporal.io/api/cloud/sink/v1;sink";
+
+import "google/protobuf/timestamp.proto";
+
+message ExportSinkSpec {
```

Comment:
Consider a unique, user-provided identifier for a sink (e.g. a name) if you want to support updating of these inside a list. Not required of course, but can be easier on you, the UI, and the user.